coalib/bears/meta.py

```diff
@@ -0,0 +1,25 @@
+from collections import defaultdict
+
+
+class bearclass(type):
+    """
+    Metaclass for :class:`coalib.bears.Bear.Bear` and therefore all bear
+    classes.
+
+    Pushing bears into the future... ;)
+    """
+
+    # by default a bear class has no aspects
```

Comment:
why not use Root aspect as default aspect for bear class. So when user wants to run bears explicitly, all the aspects covered by bear will run as Root aspect is parent aspect of every aspect.